recipes/install.rb

```diff
@@ -313,6 +314,26 @@
               }
             }
           },
+          'airflowPool' => {
```

Comment:
If I remember correctly, during an update this part is not executed.

Reply:
Code is there but since I can't test it on time, it will be merged on another commit.